### DISCUSSION TITLE
v3.2.74

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#12698] [General]
## What's Changed
* Slovenian translation by @BlazL in https://github.com/filamentphp/filament/pull/12642
* Improving Thai language translation add adding missing translation. by @phattarachai in https://github.com/filamentphp/filament/pull/12635
* Update ClearCachedComponentsCommmand Description by @JakeVdub in https://github.com/filamentphp/filament/pull/12630
* feat: loading section customizable height by @hpiaia in https://github.com/filamentphp/filament/pull/12627
* Adjust description text to the trending down arrow by @begueradj in https://github.com/filamentphp/filament/pull/12650
* Update 08-global-search.md by @Umar-Farooq-Shafi in https://github.com/filamentphp/filament/pull/12653
* Inline global search field suffix by @zepfietje in https://github.com/filamentphp/filament/pull/12656
* Norwegian translation update and fix by @Tor2r in https://github.com/filamentphp/filament/pull/12682
* Bump trix from 1.3.1 to 2.1.1 by @dependabot in https://github.com/filamentphp/filament/pull/12678
* Add missing support for extra item actions in simple repeater by @lamberttraccard in https://github.com/filamentphp/filament/pull/12674
* Check login() method exists, punt to setUser() by @cheesegrits in https://github.com/filamentphp/filament/pull/12685
* update: Japanese translation update and fix by @bzy107 in https://github.com/filamentphp/filament/pull/12695
* document: Adding command to publish missing translation files by @bzy107 in https://github.com/filamentphp/filament/pull/12696
* Add afterInit hook by @assertchris in https://github.com/filamentphp/filament/pull/12694

## New Contributors
* @BlazL made their first contribution in https://github.com/filamentphp/filament/pull/12642
* @JakeVdub made their first contribution in https://github.com/filamentphp/filament/pull/12630
* @hpiaia made their first contribution in https://github.com/filamentphp/filament/pull/12627
* @begueradj made their first contribution in https://github.com/filamentphp/filament/pull/12650
* @assertchris made their first contribution in https://github.com/filamentphp/filament/pull/12694

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.73...v3.2.74

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.74'>v3.2.74</a>.</em>

**Comment by atmonshi** [comment]
something off :)
after update to 74
<img width="757" alt="Screenshot 2024-05-08 at 8 05 47 PM" src="https://github.com/filamentphp/filament/assets/1952412/cc6aac67-673d-40ce-9fbc-d634743ffb23">

before update on 73
<img width="757" alt="Screenshot 2024-05-08 at 8 02 30 PM" src="https://github.com/filamentphp/filament/assets/1952412/a031f767-6b2d-470a-9914-79c6d3383c6f">

even with `npm run dev`

**Reply by danharrin** [reply]
Hmm! Thats strange, any other reports by other people?

**Reply by leandrocfe** [reply]
same here..

**Reply by danharrin** [reply]
I wont be home for another 2 hours, is it possible to track down what caused it if possible please?

**Reply by leandrocfe** [reply]
Yes, let me try

**Reply by leandrocfe** [reply]
There is an issue when Filament generates `forms.css`...

**Reply by fatihdurmaz** [reply]
I have the same problem
markdown-editor.js
forms.css


<img width="817" alt="Ekran Resmi 2024-05-08 23 32 53" src="https://github.com/filamentphp/filament/assets/71706970/a4756fbd-8da2-46a1-af86-3f1b34c9c222">

**Reply by danharrin** [reply]
I've just released .75 which should fix this
